README.md

```diff
@@ -40,132 +39,118 @@ Create a new .env file for the deployment based on the dotenv file included.
+## Generating SSL certificates signed by Lets encrypt.
+Some features on Greenlight and Keycloak will **NOT** work without having a secure HTTPS connection.
+To simplify the deployment process because of that requirement we provided a simple automated script `init-letsencrypt.sh` to generate Letsencrypt certificate.
+This deployment will also be in charge of handling the renewal of certificates seamlessly.
 
+When using the `init-letsencrypt.sh` script, you should add an email address to be registered when issuing the certificates.
+Kindly add your email in **.env**.
 ```
-cp -R /etc/letsencrypt/config/* data/certbot/config
+LETSENCRYPT_EMAIL=
 ```
 
-### Automated (on machines on the cloud holding a public IP and linked to a valid hostname)
+Kindly run the script with sufficient priviledges:
 ```
 ./init-letsencrypt.sh
 ```
+You will run the script in its interactive mode and therefore you'll be prompted for confirmation of some actions. 
+To run the script in a none interactive mode, please run:
+```
+./init-letsencrypt.sh -n
+```
+To replace any exisiting certificates, please run:
+```
+./init-letsencrypt.sh -n -r
+```
 
-Start the services.
+The script initially will run in staging mode to avoid reaching the rate limit.
+After a succssful staging, kindly run:
+```
+sed -i "s/LETSENCRYPT_STAGING=1/LETSENCRYPT_STAGING=0/" .env
+```
+And re-run the script:
+```
+./init-letsencrypt.sh -n -r
+```
 
-docker-compose up -d
+Congrats on your certificates!
 
-Now, the greenlight server is running, but it is not quite yet ready. The database must be initialized.
+Now start the services.
 
-For greenlight:
 ```
-docker exec -i greenlight bundle exec rake db:setup
+docker compose up -d
 ```
 
-For keycloak:
-```
-docker exec -it postgres psql -U postgres -W
-password
-CREATE DATABASE keycloakdb
-Ctrl-d
-```
+Now both Greenlight and Keycloak should be accessible through their FQDN via HTTPS!
+# Development:
 
 For development, we can use the local template by editing the variable `SITES_TEMPLATE=local` in `.env`. This will redirect the requests to your local rails app instead of doing so to the docker container.
 
 Keep in mind that the port in the template is hard-coded to `3000`, but it can be edited (or sed)  directly at `data/nginx/sites.template-local`.
 
 E.g.
 ```
-sed -i "s/NGINX_HOSTNAME:.*/NGINX_HOSTNAME:5000/" data/nginx/sites.template-local
+sed -i "s/DOMAIN_NAME:.*/DOMAIN_NAME:5000/" data/nginx/sites.template-local
 ```
 
-When keycloak is initialized, it also needs to be configured by adding a, and then set those values into the `data/greenlight/.env` file.
+When keycloak is initialized, it also needs to be configured by setting those values into the `data/greenlight/.env` file.
 
-The env variables that need to be added are:
+The ENV variables that need to be set are:
 ```
 OPENID_CONNECT_CLIENT_ID=
 OPENID_CONNECT_CLIENT_SECRET=
 OPENID_CONNECT_ISSUER=
 OPENID_CONNECT_REDIRECT=
 ```
 
-## Setup
```

Comment:
Removed in favor of https://docs.bigbluebutton.org/greenlight_v3/gl3-keycloak.html,
The readme will document the required steps to deploy Greenlight,